monkestation/code/modules/surgery/organs/augments.dm

```diff
@@ -19,7 +19,7 @@
 
 /obj/item/organ/internal/cyberimp/brain/linked_surgery/proc/on_step_completion(mob/living/user, datum/surgery_step/current_step, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results)
 	SIGNAL_HANDLER
-	if(CHECK_BITFIELD(organ_flags, ORGAN_FAILING))
```

Comment:
why these changes?

Reply:
Upstream removed that macro afaik, and if I ever end up embarking on the upstream merge project, this'll be one less thing I have to fixup.